src/gridcoin/researcher.cpp

```diff
@@ -14,6 +14,7 @@
 #include "gridcoin/researcher.h"
 #include "gridcoin/support/xml.h"
 #include "gridcoin/tally.h"
+#include "gridcoin/scraper/scraper.h"
```

Comment:
`nActiveBeforeSB` is defined in the header. By including it here, the value won't change if the protocol entry changes. To do it that way, the header should declare the variable instead (or use a standalone declaration):

```c++
extern unsigned int nActiveBeforeSB;
```

Reply:
Not entirely sure why the linker didn't complain about the ODR.

Reply:
You're right of course. Brain dead today.

Reply:
It wouldn't have changed with your code either... :)

Reply:
Heh, probably compiles to the same code too :)

Reply:
Yep. :)